src/Struct/AgenticCommerceV1/PayPalAgentError.php

```diff
@@ -0,0 +1,76 @@
+<?php declare(strict_types=1);
+/*
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Shopware\PayPalSDK\Struct\AgenticCommerceV1;
+
+use Shopware\PayPalSDK\Struct\Struct;
+
+/**
+ * @internal
+ */
+final class PayPalAgentError extends Struct
```

Comment:
I mean, it's not necessary and most of the structs aren't:
```suggestion
class PayPalAgentError extends Struct
```